spring-integration-jdbc/src/main/sql/schema.sql.vpp

```diff
@@ -52,3 +52,10 @@ CREATE TABLE INT_CHANNEL_MESSAGE (
 
 CREATE INDEX INT_CHANNEL_MSG_DATE_IDX ON INT_CHANNEL_MESSAGE (CREATED_DATE, MESSAGE_SEQUENCE);
 CREATE INDEX INT_CHANNEL_MSG_PRIORITY_IDX ON INT_CHANNEL_MESSAGE (MESSAGE_PRIORITY DESC, CREATED_DATE, MESSAGE_SEQUENCE);
+
+
+CREATE TABLE INT_METADATA_STORE  (
+	METADATA_KEY ${VARCHAR}(255),
+	METADATA_VALUE ${VARCHAR}(255),
```

Comment:
I'm not sure in so short values for both fields. The typical use-case with the `FileListFilter` might end up with pretty long path for the file.
I agree that 4000 isn't good for the primary key anyway.
But what is the choice do we have?...
Let's make 4000 for `value` at least!

The bigger length for `key` can be done in the target application during they schema contribution.